package.json

```diff
@@ -19,6 +19,7 @@
         "clean-dist": "rimraf dist/*",
         "build-esm": "tsc --project tsconfig.esm.json",
         "build-commonjs": "tsc --project tsconfig.commonjs.json",
+        "build-copy": "cp -r src/generated/grpc/ dist/esm/generated/grpc/ && cp -r src/generated/grpc/ dist/cmd/generated/grpc/",
```

Comment:
whay is this? seems not used

Reply:
files should be in dist and when build without that, they wasn"t include